docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md

```diff
@@ -56,11 +56,11 @@ Two steps remain, then:
 2.  Actually building the Cardano SL node (or, most likely, simply obtaining it
     from the IOHK's binary caches) can be performed by building the attribute `cardano-sl-static`:
 
-        $ nix-build -A cardano-sl-static --cores 0 --max-jobs 2 --no-build-output --out-link cardano-sl-1.0
+        $ nix-build -A cardano-sl-static --cores 0 --max-jobs 2 --no-build-output --out-link master
```

Comment:
Apparently `cardano-sl-static` is now `cardano-sl-node-static` (judging by #2061). It's not related to this PR, but can you fix it as well please? :)